front/app/modules/commercial/user_custom_fields/citizen/components/UserCustomFieldsSignUpInModal.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { memo, useState, useCallback, useEffect } from 'react';
```

Comment:
This was just a rename of the above file, which moved props to the next line and now git thinks it's a new file.